### DISCUSSION TITLE
Top Quark Detection with Deep Learning and Big Data (Project 238)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#74] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 238](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/blob/main/projects/Top%20Quark%20Detection%20with%20Deep%20Learning%20and%20Big%20Data/README.md)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by VaidikS124** [comment]
Hi, I am a masters student at Department of Physics, Birla Institute of Technology and Science Pilani, India and highly looking forward in working and collaborating on this project.

**Comment by sujithsidh** [comment]
Hello vaidik nice to meet u, I am final year B.E cse student. Nice to meet u

**Comment by cetuck** [comment]
Hello everyone, my name is Carl and I am a senior computer engineering student at Clemson University.

**Comment by maneetchatterjee** [comment]
Hello everyone, I am Maneet Chatterjee Department of Mechanical Engineering, IIEST.

**Comment by aarxshi** [comment]
Hey everyone! I'm a 3rd year computer science student at DSCE
